dbt/models/sale/docs.md

```diff
@@ -1,19 +1,67 @@
+# flag
+
 {% docs flag %}
-This table holds the flag information from the sales val program.
+PIN-level sales validation flags created by
+[model-sales-val](https://github.com/ccao-data/model-sales-val).
+
+This is the primary sales validation output table. Flags within this table
+should be possible to reconstruct using the other sales validation tables:
+`sale.group_mean`, `sale.parameter`, and `sale.metadata`.
+
+**Primary Key**: `meta_sale_document_number`, `run_id`, `version`
+{% enddocs %}
+
+# foreclosure
+
+{% docs foreclosure %}
+Foreclosure data ingested from Illinois Public Records (RIS).
+
+**Primary Key**: `pin`, `document_number`
 {% enddocs %}
 
+# parameter
+
 {% docs parameter %}
-This table holds information about the specifications used to flag outliers in the sales val program.
+Parameters used for each run of
+[model-sales-val](https://github.com/ccao-data/model-sales-val),
+including the statistical bounds, groupings, window sizes, etc.
+
+**Primary Key**: `run_id`
 {% enddocs %}
 
+# group_mean
```

Comment:
I've added markdown headers to the doc assets just to make them easier to read, navigate, and fold.